great-idea-website/css/index.css

```diff
@@ -63,4 +63,106 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.main-container {
+	width: 85%;
+	margin: auto;
+}
+
+nav {
+	display: inline-block;
+	width: 90%;
+	height: 40px;
+}
+
+nav a {
+	display: inline-block;
+	text-decoration: none;
+	color: grey;
+	padding-bottom: 20px;
+	align-self: center;
+}
+
+nav a:visited {
```

Comment:
interesting use of the pseudoselector.  good find on that. and good use of pseduoselectors.